src/Scorecard.js

```diff
@@ -0,0 +1,55 @@
+var Scorecard = function(){
+this.Total = null;
+this.rolls = [];
+this.frameTotals = []
+};
+
+Scorecard.prototype.roll = function(roll1, roll2){
+  if (this.rolls.length > 10){throw new Error("You cannot play more than 10 frames!");};
+  if(roll1 > 10){throw new Error("can't score more than 10!");};
+  if(roll2 > 10){throw new Error("can't score more than 10!");};
+  if((roll1+roll2) > 10) {throw new Error("You cannot score more than 10 from two rolls!");};
+  this.rolls.push([roll1,roll2]);
+};
+
+Scorecard.prototype.frameTotal = function(i){
+  var f1r1 = (this.rolls[i-1][0]);
```

Comment:
I feel like this function breaks SRP - try to keep your functions a little more focussed - they should do one thing well.